content/quests/events/demevent_rebel_arms_deal.lua

```diff
@@ -43,154 +43,243 @@ QDEF:AddConvo()
     :ConfrontState("CONFRONT", function() return true end)
         :Loc{
             DIALOG_INTRO = [[
-                * [p] You encountered two people arguing.
+                * You would think trying to sell contraband requires you be quiet. Hush Hush. Generally not try to draw attention.
+                * Either the Jake is new at the job or the Professional Rabble Rouser wants {jakes.hisher} manager.
+                jakes:
+                    !right
                 rise:
                     !left
+                    Are you against Democracy itself? We need the weaponry to protect our revolution!
                 jakes:
-                    !right
-                    No! You can't have those weapons unless you pay for them!
+                    Democracy isn't going to pay the customs fees! This is still a capitalist society, and you still need to pay!
```

Comment:
I like the interpretation that the rise is buying weapons because they are afraid that democracy won't work out and wants to have a backup plan instead of my original idea of the rise buying weapons because they straight up don't believe democracy will work.
Not sure if two people randomly dealing with contraband weapons randomly discussing politics using "academic" words fits here or not, though.

Reply:
I wasn't thinking academic words but that is true. I'd assume the Rise member is the one saying those big words and the jake is just throwing it back in their face. Maybe i'll dumb down those terms, something that two people still scraping by might actually say.